packages/firebase_messaging/android/src/main/java/io/flutter/plugins/firebasemessaging/FirebaseMessagingPlugin.java

```diff
@@ -41,9 +44,17 @@
   public static void registerWith(Registrar registrar) {
     final MethodChannel channel =
         new MethodChannel(registrar.messenger(), "plugins.flutter.io/firebase_messaging");
+    final MethodChannel backgroundCallbackChannel =
+        new MethodChannel(
+            registrar.messenger(),
+            "plugins.flutter.io/android_fcm_background",
+            JSONMethodCodec.INSTANCE);
```

Comment:
Just curious, why do we use `JSONMethodCodec` here instead of the default?

Reply:
I think this is just a artifact from the alarm manager plugin I don't think it is necessary, the default codec should be sufficient.